baseq2/maps/cameras/lab.txt

```diff
@@ -0,0 +1 @@
+(-2008.9, 678.2, 206.1) (0.001, -0.810, -0.586)
```

Comment:
the to original `cameras.txt` had a comment for the "lab" map.

Reply:
oops, thanks